src/retrace/retrace.py

```diff
@@ -2032,6 +2031,10 @@ def convert_flattened_format(self) -> bool:
             log_error("Failed to convert flattened vmcore %s - errno(%d - '%s')" %
                       (self._vmcore_path, e.errno, e.strerror))
             return False
+        except TimeoutExpired:
+            log_error("WARNING: command: '{}' exceeded {} "
+                      " second timeout - damaged vmcore?".format(' '.join(cmd), t))
```

Comment:
Extreme nitpick: let's keep the quotes consistent, please.

```suggestion
                      " second timeout - damaged vmcore?".format(" ".join(cmd), t))
```

(Repeat twice in the chunks below.)

Reply:
Ah, ok sure